api/admin/controller.py

```diff
@@ -183,25 +215,41 @@ def redirect_after_sign_in(self):
             return self.error_response(INVALID_ADMIN_CREDENTIALS)
         else:
             admin = self.authenticated_admin(admin_details)
-            flask.session["admin_email"] = admin_details.get("email")
-
-            # A permanent session expires after a fixed time, rather than
-            # when the user closes the browser.
-            flask.session.permanent = True
```

Comment:
This was moved into authenticated_admin since it's the same for both authentication mechanisms.